ush/build_GDASApp.sh

```diff
@@ -0,0 +1,41 @@
+#!/bin/bash
+# build_GDASApp.sh
+# 1 - determine if on supported host, load modules if so
+# 2 - run ecbuild
+# 3 - build
+# 4 - optional, run unit tests
+
+set -eux
+
+dir_root="$( cd "$( dirname "${BASH_SOURCE[0]}" )/.." >/dev/null 2>&1 && pwd )"
+
+dir_modules=$dir_root/modulefiles
+
+if [ ${BUILD_TARGET:-} = "orion" -o ${BUILD_TARGET:-} = "hera" ]; then
+    set +x
+    source $MODULESHOME/init/sh
+    module purge
+    module use $dir_modules
+    module load GDAS/${BUILD_TARGET}
+    module list
+    set -x
+fi
+
+# remove/create directories
+[ -d $dir_root/bin ] || mkdir -p $dir_root/bin
+
+rm -rf $dir_root/src/build
```

Comment:
Just curious.  UFSDA built in `$dir_root/build`.   Why build GDASApp in `$dir_root/src/build`?

Reply:
I wanted to distinguish between building the "App" and building "JEDI". I suspect in the future these will be one and the same with cmake (I assume that is @aerorahul 's vision), but it's not there yet. But I don't know if it matters all that much and can change it if you'd like.

Reply:
Thanks, @CoryMartin-NOAA , for the explanation.  I'm not advocating any changes.  I was just curious.